src/test/java/org/kainos/ea/validator/RegisterValidatorTest.java

```diff
@@ -3,45 +3,45 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.kainos.ea.cli.Register;
-import org.kainos.ea.core.RegisterValidator;
+import org.kainos.ea.core.Validator;
 
 import static org.junit.jupiter.api.Assertions.*;
 import static org.kainos.ea.cli.Role.Admin;
 import static org.kainos.ea.cli.Role.Employee;
 
 public class RegisterValidatorTest {
```

Comment:
this file can be removed as its duplicate of src/test/java/org/kainos/ea/service/ValidatorServiceTest.java